tests/AutoReview/TravisTest.php

```diff
@@ -1,184 +0,0 @@
-<?php
-
-/*
- * This file is part of PHP CS Fixer.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace PhpCsFixer\Tests\AutoReview;
-
-use PhpCsFixer\Preg;
-use PhpCsFixer\Tests\TestCase;
-use PhpCsFixer\Tokenizer\Tokens;
-use PHPUnit\Framework\Constraint\TraversableContains;
-use Symfony\Component\Yaml\Yaml;
-
-/**
- * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
- *
- * @internal
- *
- * @coversNothing
- * @group auto-review
- * @group covers-nothing
- */
-final class TravisTest extends TestCase
```

Comment:
:-1: for dropping
i would rather expecting to have those tests migrated to read github actions config than fully dropped.

Reply:
Do you consider this a blocker for merging. I feel like the GitHub UI already makes it pretty clear if stuff is missing, with the human readable job names I have gone for.

Reply:
yes, I do

but pls, align with `@sanmai` to prepare single PR instead of putting more effort in one or another PR